k8s/openebs-operator.yaml

```diff
@@ -155,6 +159,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace 
```

Comment:
We have to add Namespace downward API to snapshot-controller and snapshot-provisioner deployment containers too.